aws_grant_user_access/src/process_event.py

```diff
@@ -84,40 +95,15 @@ def publish_sns_message(message: SNSMessage) -> None:
         )
 
 
-def _validate_role(role_arn: str) -> bool:
-    for role in PERMITTED_ROLES:
-        if role.lower() in role_arn.lower():
-            return True
-    return False
-
-
-def _validate_user(user_name: str) -> bool:
+def is_a_platform_engineer(username: str) -> bool:
```

Comment:
Should this be accepting a list of strings (usernames) instead of a single username?